tests/test_sails.py

```diff
@@ -0,0 +1,231 @@
+import pytest
+import requests
+import json
+from pymongo import MongoClient
+
+
+"""
+Basic Queries
+"""
+
+query = {
+    'firstName' : 'test_use',
+    'lastName' : 'test01',
+    'email' : 'test@capstonedd.com',
+    'type' : 'buyer'
+  }
+  
+findq = {
+    'firstName' : 'test_use'
+  }
+  
+testq = {
+    'lastName' : 'test02'
+  }
+  
+dataquery = json.dumps(query)
+datafquery = json.dumps(findq)
+
+"""
+Connect to MongoDB
+"""
+
+client = MongoClient('capstonedd.cs.pdx.edu', 27017)
+# Uncomment this line if you have auth options
+# client = MongoClient('mongodb://admin:teamG@capstonedd.cs.pdx.edu')
+collection = client.test.users
+# Test if test record exists
+c = collection.count(findq)
+if c > 0:
+  collection.delete_many(findq)
+
+
+"""
+TEST RESPONSES
+Includes all status code responses for both get and post requests.
+"""
+
+def test_get_no_param_200_code():
+  """
+  SUMMARY:  Test that GET method returns a 200 code ('ok')
+  METHOD:   Send a GET request with no parameteres.
+  FAIL:     Status code is not 200.
+  """
+  
+  # Act
+  r = requests.get('http://capstonedd.cs.pdx.edu:1337/users')
+
+  # Assert
+  assert r.status_code == 200
+  
+  
+def test_post_returns_200_code():
+  """
+  SUMMARY:  Test that valid POST body returns a 200 code ('ok')
+  METHOD:   Send a POST request with proper post body.
+  FAIL:     Status code is not 200 or nothing has been inserted.
+  """
+  
+  # Act
+  r = requests.post('http://capstonedd.cs.pdx.edu:1337/users', data=dataquery)
+
+  # Assert & redo
+  c = collection.count(findq)
+  if c > 0:
+    collection.delete_many(findq)
+    assert r.status_code == 200
+  else:
+    raise AssertionError
```

Comment:
I'm actually not sure what we're testing here. I thought we were testing that a POST returned a 200. But it also looks like you're testing if the POST data was captured in the DB. This should be another test.